pkg/config/validation/validation.go

```diff
@@ -819,6 +819,13 @@ func validateOutlierDetection(outlier *networking.OutlierDetection) (errs error)
 	if outlier.ConsecutiveErrors < 0 {
 		errs = appendErrors(errs, fmt.Errorf("outlier detection consecutive errors cannot be negative"))
```

Comment:
probably need this same check for the new fields?

Reply:
Yes, none of them should be negative. Also, maybe a warning if 5xxErrors < GatewayErrors would also be good?

Reply:
Ah, the new fields are UInt, so negative values are already handled.